lib/fetcher.js

```diff
@@ -101,13 +102,32 @@ module.exports = heroku => {
     return attachments.filter(a => a.addon.plan.name.startsWith('heroku-postgresql'))
   }
 
+  function * getAttachmentNamesByAddons (attachments) {
+    let results = {}
+    attachments.forEach(function (a) {
+      let attachmentNames = results[a.addon.id] || []
+      attachmentNames = attachmentNames.concat(a.name)
+      results[a.addon.id] = attachmentNames
+    })
+
+    return results
```

Comment:
This is fine, but you could simplify this to

```javascript
return attachments.reduce((results, a) => {
  results[a.addon.id] = (results[a.addon.id] || []).concat(a.name)
  return results
}, {})
```

or even jus tdrop line 109 and move the `.concat(a.name)` to 110.